### ISSUE TITLE
[Swift 5.0]: LowerACLThanParentRule is broken

### ISSUE THREAD

**Issue by jpsim** [realm/SwiftLint#2607]
```swift
private class Foo { func bar() {} }
                    ^ warning: Lower ACL than parent Violation: Ensure definitions have a lower access control level than their enclosing parent (lower_acl_than_parent)
```

It seems like members without an explicit ACL annotation in Swift 5 are incorrectly assumed to be `internal`. This causes incorrect violations in Swift 5 that aren't present in Swift 4.x.

**Comment by jpsim** [comment]
I filed [SR-9777](https://bugs.swift.org/browse/SR-9777) to track this.
